tsl/src/compression/compression.c

```diff
@@ -2672,6 +2676,16 @@ decompress_batches_seqscan(Relation in_rel, Relation out_rel, Snapshot snapshot,
 
 		write_logical_replication_msg_decompression_start();
 		result = delete_compressed_tuple(&decompressor, snapshot, compressed_tuple);
+		/* skip reporting error if isolation level is < Repeatable Read
+		 * since somebody decompressed the data concurrently, we need to take
+		 * that data into account as well when in Read Committed level
+		 */
+		if (result == TM_Deleted && !IsolationUsesXactSnapshot())
+		{
+			write_logical_replication_msg_decompression_end();
+			stats.batches_decompressed++;
+			continue;
+		}
```

Comment:
Should this change some test output? I.e. we would report an error for this case previousely.